imagetest/testworkflow.go

```diff
@@ -283,8 +282,14 @@ func finalizeWorkflows(ctx context.Context, tests []*TestWorkflow, zone, gcsPref
 			}
 		}
 
-		twf.wf.Sources["wrapper"] = fmt.Sprintf("%s.%s", testWrapperPath, arch)
-		twf.wf.Sources["testpackage"] = fmt.Sprintf("/%s.%s.test", twf.Name, arch)
+		if twf.osUnderTest == "windows" {
+			twf.wf.Sources["testpackage"] = fmt.Sprintf("/%s.%s.test.exe", twf.Name, arch)
+			twf.wf.Sources["wrapper.exe"] = "/wrapper.exe"
+			twf.wf.Sources["startup.ps1"] = "/startup.ps1"
+		} else {
+			twf.wf.Sources["testpackage"] = fmt.Sprintf("/%s.%s.test", twf.Name, arch)
+			twf.wf.Sources["wrapper"] = fmt.Sprintf("%s.%s", testWrapperPath, arch)
+		}
```

Comment:
this is fine but it may be a bit more readable to have a paradigm that just uses the string formatter as the decider, you can do that like:

```go
var suffix string
if twf.osUnderTest == "windows" {
    suffix = ".exe"
}
twf.wf.Sources["testpackage"] = fmt.Sprintf("/%s.%s.test%s", twd.Name, arch, exe)
```

main benefit is reducing complexity of reading. rather than two whole blocks for different OSes, OS is simply considered in the formatting step. just a suggestion

Reply:
Went ahead and did this as it does make it much cleaner.